frontend/app/templates/components/footer.hbs

```diff
@@ -1,6 +1,12 @@
+{{#if this.isChangelogOpen}}
+  <Changelog @onAbort={{this.toggleChangelogModal}} @title="Changelog"/>
```

Comment:
We don't need to translate the title here right? Probably fine

Reply:
It is the same in every language.